installers/install.ps1

```diff
@@ -147,7 +147,11 @@ $PSDefaultParameterValues['*:Encoding'] = 'utf8'
 
 $env:ACTIVESTATE_SESSION_TOKEN = $script:SESSION_TOKEN_VALUE
 & $exePath $args --source-installer="install.ps1"
+$success = $?
 if (Test-Path env:ACTIVESTATE_SESSION_TOKEN)
 {
     Remove-Item Env:\ACTIVESTATE_SESSION_TOKEN
-}
\ No newline at end of file
+}
+if ( !$success ) {
+  exit 1
```

Comment:
Is there a way for us to exit with the error code from the state tool? This is currently in line with what we were doing before. Just curious.

Reply:
Nope, doesn't look like it :( https://github.com/ActiveState/cli/runs/5891595613?check_suite_focus=true